### ISSUE TITLE
Any kind of community communication?

### ISSUE THREAD

**Issue by souze** [philippkeller/rexpect#63]
Should we add some sort of zulip/discord/slack for this package? Adding a channel in an existing forum would be the easiest, anything else feels a bit overkill I think.

**Comment by matthiasbeyer** [comment]
I don't see any benefit, to be honest.
Especially because zulip, discord and slack are gated communities. I do not participate in any.

Why do you think are issues not sufficient?

**Comment by souze** [comment]
I find it a bit easier to have a back-and-forth conversation about a topic, or just ask a fast/simple question in a more chat-like interface.
I see the point though that it's a bit overkill for this size package.

**Comment by matthiasbeyer** [comment]
I think that keeping even simple questions in the issues is better because users that might come up with the same question can easily find them here.
If you think about more personal questions between us authors, feel free to shoot an email any time! :wink:

**Comment by petreeftime** [comment]
I don't think this library is particularly heavy traffic to warrant a separate discussion platform and it makes finding information quite a bit more challenging. Keeping all discussions here is healthier IMO.

**Comment by souze** [comment]
Sounds good then :)
